js/src/lib/Details/Popularity.js

```diff
@@ -52,6 +52,28 @@ const Dependents = ({ dependents, humanDependents, name }) =>
     />
   );
 
+const formatHits = (hits) => {
+  if (hits >= 1e9) {
+    return (Math.round(hits / 1e7) / 100) + "b";
+  } else if (hits >= 1e6) {
+    return (Math.round(hits / 1e4) / 100) + "m";
+  } else if (hits >= 1000) {
+    return (Math.round(hits / 10) / 100) + "k";
+  }
+
+  return hits;
+};
+
+const JsDelivrHits = ({jsDelivrHits}) => (<Di
+  icon="downloads"
+  title={window.i18n.detail.jsdelivr_hits}
+  description={
+    <span title={jsDelivrHits.toLocaleString(window.i18n.active_language)}>
```

Comment:
Does this work for all languages supposed by the Yarn site?

Reply:
yes, this worked and surprisingly for really old browsers too